manifests/overlays/additional/jupyter-amd-pytorch-notebook-imagestream.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/opendatahub-io/notebooks/blob/main/jupyter/amd/pytorch"
+    opendatahub.io/notebook-image-name: "AMD-PyTorch"
+    opendatahub.io/notebook-image-desc: "Jupyter AMD optimized PyTorch notebook image for ODH notebooks."
+    opendatahub.io/notebook-image-order: "12"
+  name: jupyter-amd-pytorch
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+    # N Version of the image
+    - annotations:
+        opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.9"},{"name":"AMD-PyTorch","version":"2.3"}]'
+        opendatahub.io/notebook-python-dependencies: '[{"name":"AMD-PyTorch","version":"2.3"},{"name":"Tensorboard","version":"2.16"},{"name":"Kafka-Python","version":"2.0"},{"name":"Matplotlib","version":"3.6"},{"name":"Numpy","version":"1.26"},{"name":"Pandas","version":"1.5"},{"name":"Scikit-learn","version":"1.4"},{"name":"Scipy","version":"1.11"},{"name":"Elyra","version":"3.15"},{"name":"PyMongo","version":"4.5"},{"name":"Pyodbc","version":"4.0"}, {"name":"Codeflare-SDK","version":"0.13"}, {"name":"Sklearn-onnx","version":"1.15"}, {"name":"Psycopg","version":"3.1"}, {"name":"MySQL Connector/Python","version":"8.0"}]'
```

Comment:
Could you align this packages here with the ones that we have on pipfile here? https://github.com/opendatahub-io/notebooks/blob/f4321caac22951de2d409d9d05f869ab0212aa98/jupyter/amd/pytorch/ubi9-python-3.9/Pipfile

for example the codeflare needs to be change, take a look for the rest